client/core/notification.go

```diff
@@ -242,7 +242,8 @@ func newSendNote(topic Topic, subject, details string, severity db.Severity) *Se
 // OrderNote is a notification about an order or a match.
 type OrderNote struct {
 	db.Notification
-	Order *Order `json:"order"`
+	Order       *Order `json:"order"`
+	TemporaryID uint64 `json:"tempID,omitempty"`
```

Comment:
OK with the caveat that we can never use 0

Reply:
Yes, we can never use zero.

Reply:
Then should doc that at the site of the atomic declaration so that never happens.